toolchain/templates/default.mako

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+. "${rootdir}/toolchain/util.sh"
+
+% if engine == 'batch':
+    error "The$MAGENTA default$COLOR_RESET template does not support batch jobs. Please use a different template via the $MAGENTA--computer$COLOR_RESET option.\n"
+    exit 1
+% endif
+
+<%include file="prologue.mako"/>
+
+warn "This is the$MAGENTA default$COLOR_RESET template."
+warn "It is not intended to support all systems and execution engines."
+warn "Please use a different template via the $MAGENTA--computer$COLOR_RESET option."
+echo
+
+% for binpath in binpaths:
+    echo -e ":) Running $MAGENTA${binpath}$COLOR_RESET:\n"
+
+    % if not mpi:
+        ${' '.join([f"'{x}'" for x in profiler ])} "${binpath}"
+    % else:
+        if command -v jsrun > /dev/null; then
```

Comment:
can we not just pass `-b` in here? i know we should be using the templates but this seems like an easy improvement of sorts.

Reply:
As per above, I agree and I just added it.